.travis.yml

```diff
@@ -1,16 +1,27 @@
+sudo: false
```

Comment:
True.
According to the docs this is the default for all repos activated since 2015. And was already the default behaviour for tldextract.

Should I remove it?